Remotion/Core/Core/Logging/Log4NetLog.cs

```diff
@@ -105,7 +105,7 @@ public bool IsEnabled (LogLevel logLevel)
       return _logger.IsEnabledFor (Convert (logLevel));
     }
 
-    private LoggingEvent CreateLoggingEvent (Level level, int? eventID, object message, Exception? exceptionObject)
+    private LoggingEvent CreateLoggingEvent (Level level, int? eventID, object? message, Exception? exceptionObject)
```

Comment:
Interesting question: log4net does not support null messages, but our code would always throw in the case of an error in LogLoggingError if message was null. 

It's probably not really helpful to support a null-message, but we need to make a descicion here. either we add a null-guard to Log in line 86 and LogFormat in line 95 or we support null in the interface. For now, the cheap solution in line 150 and a TODO-ticket are probably sufficient.

Status quo says, that NULL worked unless logging itself had another error. So the non-breaking variant would be to support NULL, strange as it may seem.